e2e_samples/parking_sensors/scripts/init_environment.sh

```diff
@@ -47,5 +47,6 @@ fi
 AZURESQL_SERVER_PASSWORD=${AZURESQL_SERVER_PASSWORD:-}
 if [ -z "$AZURESQL_SERVER_PASSWORD" ]
 then 
-    export AZURESQL_SERVER_PASSWORD="$(makepasswd --chars 16)"
-fi
\ No newline at end of file
+    #Increase the complexity by appending the complex string, as some passwords are not cmplex enough.
```

Comment:
wouldn't it be easier to just ask for a longer password to the function? or make the password function return more complex passwords? we are not really increasing the complexity with this step, just making it longer

Reply:
as we will remove SQL Auth, just closing this conversation